Alert.css

```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --primary-color: #e6e6fa;
+  --primary-text-color: #490e60;
+  --secondary-color: #326771;
+  --secondary-text-color: #071203;
+  --success-color: #84f1c9;
+  --success-text-color: #1b7052;
+  --danger-color: #fcc5c5;
+  --danger-text-color: #dc3545;
+  --warning-color: #ffe7c5;
+  --warning-text-color: #ff9204;
+}
```

Comment:
you can make utility class for this

Reply:
Yes, after completeion of P0 components I will make utility class for all combined. Thanks